lib/proofer/vendor/mock.rb

```diff
@@ -109,6 +112,14 @@ def build_question_set(_vendor_resp)
 
       private
 
+      def financial_errors(financials)
+        errors = {}
+        financials.keys.each do |key|
+          errors[key] = "The #{key} could not be verified."
+        end
+        errors
```

Comment:
Thanks for the tip about `each_with_object` -- I had not seen that before.

I have enabled codeclimate for this repo. 

Neither rubocop nor reek complain about that locally for me.